docs/source/reference/util/sparse.rst

```diff
@@ -1,6 +1,10 @@
 Sparse utilities
 -----------------------------------
 
+Sparse matrix representations that support back-propagation for e.g. efficient
+sparse matrix multiplication and conversion functions between sparse and dense
+representations.
+
```

Comment:
How about adding a note like this to avoid confusing users that sparse matrix computations are available everywhere?
`Not all functions support use of sparse matrix. See F.sparse_matmul for the example usage.`

Reply:
Fixed.